taxi/commands.py

```diff
@@ -406,6 +407,8 @@ def run(self):
             )
             local_entries_list = []
             for (date, entries) in local_entries.iteritems():
+                if not_today and date == datetime.date.today():
```

Comment:
Why not use the same logic as above, ie. add an `exclude_today` parameter to `get_local_entries`?